manifests/05-dedicated-admin-operator.ClusterRole.yaml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
This is not going to work, k8s has a lock to prevent access escalation. A `ServiceAccount` dedicated-admin-operator can't assign a `RoleBinding` to a role that is more powerful than itself (dedicated-project-admin).

With that the operator will return a msg like that when trying to assign a `RoleBinding`:
`clusterroles.rbac.authorization.k8s.io "<RBAC>" is forbidden: attempt to grant extra privileges`

Reply:
The `dedicated-admin-operator` role is assigned in https://github.com/jewzaam/dedicated-admin-operator/blob/revise-permissions-srep-1008/manifests/11-dedicated-admin-operator-sa.RoleBindings.yaml#L13 which isn't managed by the operator, it's OLM that would set this up.

Reply:
Sorry, I didn't notice you assigned the admin Role to the operator.
Looks good.